### ISSUE TITLE
Split up workflow for developers and POs/PMs

### ISSUE THREAD

**Issue by katbow** [dwyl/contributing#50]
Currently the workflow contains steps for the developer creating the original issue, and the person maintaining the project. These are mixed together within the same flow and can make it difficult to figure out what your responsibilities may be.

Suggestion is to split into 2 separate workflows for the *Developer* and the *Product Owner/Project Maintainer*, so that you can quickly find the steps to follow for your role.

**Comment by iteles** [comment]
Agreed! We should have a diagram #17 that weaves the two together somehow as it's quite useful to see the ping-pong between the two.
